docs/advanced_usage.rst

```diff
@@ -241,6 +241,63 @@ next step of the pipeline or ``None`` if the trace should be discarded::
 
 (see filters.py for other example implementations)
 
+Http layer
+----------
+
+..  _http-headers-tracing:
+
+Headers tracing
+^^^^^^^^^^^^^^^
+
+
+For a selected set of integrations, it is possible to store http headers from both requests and responses in tags.
+
+Configuration can be provided both at the global level and at the integration level.
+
+Examples::
+
+    from ddtrace import config
+
+    # Global config
+    config.trace_headers([
+        'user-agent',
+        'transfer-encoding',
+    ])
+
+    # Integration level config, e.g. 'falcon'
+    config.falcon.http.trace_headers([
+        'user-agent',
+        'some-other-header',
+    ])
+
+The following rules apply:
+  - headers configuration is based on a whitelist. If a header does not appear in the whitelist, it won't be traced.
+  - headers configuration is case-insensitive.
+  - headers configuration does not allow for wildcards.
```

Comment:
```suggestion
```

This line is probably not necessary, especially if we give the people the idea that wildcards could be an option, they will ask for it. More so than if we don't mention it at all.